doc/chapters/integration-guidelines/10-write-buffers.rst

```diff
@@ -0,0 +1,24 @@
+Write buffered filesystems
+==========================
+
+To enable the write buffers for the filesystems of an app, you need to specify
+a short json object that is sent to the SoftwareContainerAgent.Create method.
```

Comment:
Keep the same way of writing:
``com.pelagicore.SoftwareContainerAgent.Create(id)``